main.go

```diff
@@ -98,20 +98,26 @@ func avsSyncMain(cliCtx *cli.Context) error {
 	for _, quorum := range cliCtx.IntSlice(QuorumListFlag.Name) {
 		quorums = append(quorums, byte(quorum))
 	}
+	syncInterval := cliCtx.Duration(SyncIntervalFlag.Name)
 	avsSync := NewAvsSync(
 		logger,
 		avsReader,
 		avsWriter,
 		cliCtx.Duration(SleepBeforeFirstSyncDurationFlag.Name),
-		cliCtx.Duration(SyncIntervalFlag.Name),
+		syncInterval,
 		operators,
 		quorums,
 		cliCtx.Bool(FetchQuorumDynamicallyFlag.Name),
 		cliCtx.Int(retrySyncNTimes.Name),
 		cliCtx.Duration(ReaderTimeoutDurationFlag.Name),
 		cliCtx.Duration(WriterTimeoutDurationFlag.Name),
 	)
-	avsSync.Start()
 
+	if syncInterval == 0 {
+		logger.Infof("Sync interval is 0, running updateStakes once and exiting")
+		return avsSync.updateStakes()
+	}
```

Comment:
I'd personally prefer if we moved this logic inside Start. Basically just don't run the `for range ticket.C` if syncInterval is set to 0.
This way we keep the logic all at the same place (so if people ever add logic to Start(), they won't forget to also update this other entrypoint). wdyt?